be/src/olap/column_predicate.h

```diff
@@ -69,6 +69,8 @@ class ColumnPredicate {
     virtual void evaluate_vec(vectorized::IColumn& column, uint16_t size, bool* flags) const {};
     uint32_t column_id() const { return _column_id; }
 
+    virtual bool is_in_predicate() { return false; }
```

Comment:
is_in_predicate should be declared as const because it seems read-only

Reply:
👌🏻